test/sslapitest.c

```diff
@@ -3899,7 +3905,10 @@ static int early_data_skip_helper(int testtype, int cipher, int idx)
     unsigned char buf[20];
     size_t readbytes, written;
 
-    if (is_fips && cipher == 4)
+    if (is_fips && cipher >= 4 )
+        return 1;
+
+    if (ciphersuites[cipher] == NULL)
         return 1;
```

Comment:
We should probably do this instead `return TEST_skip("explanation here");`

Reply:
Now returning `TEST_skip("Cipher not supported")`.